docs/decisions/0007-executive-education-2U-terms-policies-user-profile-data-route.md

```diff
@@ -0,0 +1,74 @@
+# 0007. Add page route for gathering consent for terms/policies and user profile data for Executive Education (2U) courses
+
+## Status
+
+Accepted (08-01-2022)
+
+## Context
+
+As part of our Product Line Integration work (PLI) to make 2U content (i.e., Executive Education courses) accessible to customers with an external Learning Management System (LMS), we will need to collect required information from the user in order for fulfillment of Executive Education (2U) courses.
+
+The fulfillment of Executive Education (2U) courses is handled by the GetSmarter Enterprise API Gateway (GEAG), in which edX for Business is considered an external service provider to GetSmarter. 
+
+The GEAG `/allocations` API endpoint denotes the following required information from the user:
+
+* Reading and agreeing to the Terms and Policies provided by GEAG
+* Collecting required user profile:
+  * Address Line 1
+  * Address Line 2
+  * City
+  * Postal code
+  * Country
+  * Country code
+  * First name
+  * Last name
+  * Email
+  * Date of birth
+
+We may be able to programmatically get some of this user profile information from the Open edX LMS (e.g., full name, country code, email), but we will still need to build a UI to collect the information we do not have (e.g., address, date of birth, etc.) within Open edX and a way for the user to view and agree to the terms and policies.
```

Comment:
Decision: For the MVP we will have the learner supply all this data.

Reply:
Good callout, will update.